Backend/Core/Infrastructure.Business/Users/UserService.cs

```diff
@@ -18,29 +18,30 @@ public UserService(IUserRepository<T> repository, IIdentityUserService identityU
 
         public IIdentityUserService IdentityUserService { get; }
 
-        public abstract string[] GetSpecificRoles();
+        public abstract Task<string[]> GetSpecificRoles();
 
         public async Task<T> Create(string firstName, string lastName, string phoneNumber)
```

Comment:
We create entity by identity user. The key for creating is phone number. I think name "CreateByPhoneNumber" more suitable for this situation.